### ISSUE TITLE
Proto Search field for Proto switch

### ISSUE THREAD

**Issue by lmangani** [sipcapture/homer-ui#330]
It seems currently not possible to switch proto/transaction type for existing Proto Search forms. 
Ideally, this field could be part of the available form fields to allow switching between contexts faster like in HEPIC proto search.

![image](https://user-images.githubusercontent.com/1423657/97641795-02ae2e80-1a44-11eb-95af-06cc492f3354.png)

**Comment by adubovikov** [comment]
we have added it but with some limitation. The fields thatt will be forced in use from one profile in another should be same. I.e. have created a widget base on 1_call , added the new field "profile" and force search in "registration" in this case only the same field can be applied in search: method, source/destination ip/search, proto, capture id, but some custom fields that have been added only to the call profile will generate empty search for registration or default because they are not exists. Ie. X-CID....


![Screenshot from 2020-10-30 18-52-42](https://user-images.githubusercontent.com/4513061/97747544-6be97c80-1aec-11eb-9d43-9d2410e42a11.png)

![Screenshot from 2020-10-30 18-52-51](https://user-images.githubusercontent.com/4513061/97747551-6e4bd680-1aec-11eb-8c22-a49008afc270.png)

and the mapping schema for existing setups:

```
 {
    "id": "profile",
    "name": "Profile",
    "type": "string",
    "index": "none",
    "form_type": "select",
    "form_default": [
        "call",
        "registration",
        "default"
    ],
    "position": 23,
    "skip": false,
    "hide": true,
    "profile": true
  },

```
